git/helpers.sh

```diff
@@ -0,0 +1,59 @@
+#
+# Deletes a branch locally and remotely
+#
+function gitdelete() {
```

Comment:
Can we update this to be similar to `gitmerge`, such that the git commands reference variables rather than $1 and $2?

Reply:
sure

Reply:
Done